packages/builder/src/components/design/settings/controls/FormStepConfiguration.svelte

```diff
@@ -92,7 +92,8 @@
   }
 
   const addStep = () => {
-    value = value.toSpliced($currentStep + 1, 0, {})
+    const newInstance = componentStore.createInstance(componentType)
+    value = value.toSpliced($currentStep + 1, 0, newInstance)
```

Comment:
Instead of using an empty object, we now create a fake component instance for each form step. This is how we usually handle these settings, and makes it easier to use later with generic utilities because we have a real component type saved for which we can lookup the definition and settings.